Backend/main.go

```diff
@@ -76,7 +78,13 @@ func main() {
 	authorized.GET("/api/users", handlers.GetUsersHandler)
 	authorized.GET("/api/user/:id", handlers.GetOneUserHandler)
 	authorized.GET("/api/username/:username", handlers.GetOneUserbyUsernameHandler)
-	authorized.PUT("/api/user/addNewUser", handlers.AddNewUserHandler)
+	authorized.POST("/api/user/addNewUser", handlers.AddNewUserHandler)
+	authorized.PATCH("/api/user/editFullName/:newName/:id", handlers.EditNameHandler)
+	authorized.PATCH("/api/user/editBio/:newBio/:id", handlers.EditBioHandler)
+	authorized.PATCH("/api/user/editUsername/:newUsername/:id", handlers.EditUsernameHandler)
```

Comment:
and here, not that it matters, but instead of putting :id, it could be :username, although technically golang doesn't really care, it's just a placeholder for anything, but if we do change it to username, it'd be a bit more descriptive!